src/UpdateCars.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Please remove this file from the story1 branch - we don't ever want to merge code that isn't used into main